# goal definition
std_msgs/Float64MultiArray goal
std_msgs/Float64MultiArray errorWeights
std_msgs/Float64MultiArray mpcWeights
std_msgs/Float64 maxError
std_msgs/Float64 safetyMargin
---
#result definition
bool success
std_msgs/Float64MultiArray finalState
std_msgs/Duration executionTime
---
#result feedback
int32 errorFlag
